cli/js/web/headers.ts

```diff
@@ -43,15 +44,23 @@ function validateValue(value: string): void {
 class HeadersBase {
   [headerMap]: Map<string, string>;
 
+  // https://tools.ietf.org/html/rfc6265#section-4.1.1
+  // Servers SHOULD NOT include more than one Set-Cookie header field in
+  // the same response with the same cookie-name
+  [cookieMap]: Map<string, string>;
+
```

Comment:
In general Headers are not a Key/Value pair - I think headerMap should be replaced with a list of key, value pairs.

Reply:
Implemented a `Map<cookieName, cookieValue>` so we only kept the last value for the same cookie name, which is what the spec says. If I use a list of key/value pairs, I'll need to add additional logic to remove duplicates and keep the last one.